avocado/utils/software_manager.py

```diff
@@ -57,7 +57,14 @@
 log = logging.getLogger('avocado.test')
 
 
-SUPPORTED_PACKAGE_MANAGERS = ['apt-get', 'yum', 'zypper', 'dnf']
```

Comment:
We don't need a method that returns a dictionary here, we can just have a dictionary, such as:

```diff
diff --git a/avocado/utils/software_manager.py b/avocado/utils/software_manager.py
index 6e285769..635b7127 100644
--- a/avocado/utils/software_manager.py
+++ b/avocado/utils/software_manager.py
@@ -56,17 +56,6 @@ from . import path as utils_path
 
 log = logging.getLogger('avocado.test')
 
-
-def _get_backends():
-    """
-    Get the mapping of package manager name to implementation class.
-
-    Note: New implementations should have an entry added in the map.
-    """
-    return {'apt-get': AptBackend, 'yum': YumBackend, 'dnf': DnfBackend,
-            'zypper': ZypperBackend}
-
-
 class SystemInspector:
 
     """
@@ -89,7 +78,7 @@ class SystemInspector:
         to find the best supported system.
         """
         list_supported = []
-        for high_level_pm in _get_backends():
+        for high_level_pm in SUPPORTED_PACKAGE_MANAGERS:
             try:
                 utils_path.find_command(high_level_pm)
                 list_supported.append(high_level_pm)
@@ -1235,6 +1224,14 @@ def install_distro_packages(distro_pkg_map, interactive=False):
     return result
 
 
+SUPPORTED_PACKAGE_MANAGERS = {
+    'apt-get': AptBackend,
+    'yum': YumBackend,
+    'dnf': DnfBackend,
+    'zypper': ZypperBackend
+    }
+
+
 def main():
     parser = argparse.ArgumentParser(
         "usage: %prog [install|remove|check-installed|list-all|list-files|add-repo|"
```

Reply:
I second this opinion - using a module level dictionary is much simpler to read.